File: Fenicia.Module.Basic/Domains/Employee/DTOs/Responses/GetEmployeeByIdResponse.cs
using Fenicia.Module.Basic.Domains.Customer.DTOs.Responses;

namespace Fenicia.Module.Basic.Domains.Employee.DTOs.Responses;

public record GetEmployeeByIdResponse(Guid Id, Guid PositionId, Guid PersonId, string Name, string? Email, string? PhoneNumber, string? Document, AddressResponse? Address);

File: Src/Front/src/types/basic/employee/get-employee-by-id-response.ts
import { AddressResponse } from "../address/address-response";

export type GetEmployeeByIdResponse = {
    id: string;
    positionId: string;
    personId: string;
    name: string;
    email?: string;
    phoneNumber?: string;
    document?: string;
    address?: AddressResponse;
};
